tests/testthat/test-on-failure.r

```diff
@@ -0,0 +1,12 @@
+context("on-failure")
+
+test_that("on_failure should work", {
```

Comment:
Please style according to http://style.tidyverse.org (using the styler package will be easiest)

Reply:
that link is broken. I was able to find [this](http://style.tidyverse.org/tests.html), but it doesn't say anything about styling the `test_that` calls. As far as I can tell, I've structured these exactly the same as other test files in the repo, e.g. [test-assertions.R](https://github.com/hadley/assertthat/blob/master/tests/testthat/test-assertions.R).

What are you asking me to change about the style here?